fontforgeexe/fontinfo.c

```diff
@@ -8433,16 +8447,6 @@ return;
     vgcd[13].gd.u.list = ibmfamily;
     vgcd[13].creator = GListButtonCreate;
 
```

Comment:
I tried to Tab but it really only works with numeric fields, I see no outline and cannot open up list elements when tabbing. 
I'll note that there is existing disorder, for instance OS/2 Version which is the first field of this tab is ordered towards the bottom. Anyways, tab navigation isn't useable (at least on my computer) for this kind of elements.